README.md

```diff
@@ -43,9 +30,9 @@ commercetools sync is a Java library that could be used to synchronise CTP data
 
 
 Currently this library supports synchronising
- - [Categories](https://github.com/commercetools/commercetools-sync-java/tree/master/src/main/java/com/commercetools/sync/categories#commercetools-category-sync)
- - [InventoryEntries](https://github.com/commercetools/commercetools-sync-java/tree/master/src/main/java/com/commercetools/sync/inventories#commercetools-inventory-sync). (_Beta_: Not recommended for production use yet.)
-
+ - [Categories](docs/usage/CATEGORY_SYNC.md)
```

Comment:
i'd recommend to start all the paths with root separator, e.g.:
<code><b>/</b>docs</code>

in this case all the documents will refer the same directory, not relative path

Reply:
8d8a740